Assets/World/Global.gd

```diff
@@ -1,6 +1,10 @@
-tool
 extends Node
 
+enum WindowMode {
+	WINDOWED,
+	FULLSCREEN
```

Comment:
You are missing a few trailing commas in some of these, typically trailing commas are nice because they make refactoring easier, though in this particular case it doesn't matter much because it's possible there won't be any other options.